src/js/components/List/List.js

```diff
@@ -357,13 +366,22 @@ const List = React.forwardRef(
                       // accessibility guidelines that focus remains on `ul`
                       listRef.current.focus();
                     },
-                    onMouseOver: () => setActive(index),
-                    onMouseOut: () => setActive(undefined),
-                    onFocus: () => {
+                    onMouseOver: () => {
+                      setLastActive(undefined);
                       setActive(index);
+                    },
+                    onMouseOut: () => {
+                      console.log('Mouse Out');
+                      setActive(lastActive);
```

Comment:
Why are we setting `active` during `onMouseOut`?

I would have expected that we would set `active` as we already do. Then, we would `setLastActive(active)` during `onBlur`. Then, in `onFocus` we would `setActive(lastActive)`. That feels like it would mirror the behavior we are shooting for and be simple enough. But, I wonder if I'm missing something?

Reply:
After applying `onFocus and onBlur` directly to the `<StyledList>` the intended logic you described is now back in place.